base/applications/regedit/framewnd.c

```diff
@@ -164,10 +164,36 @@ void SetupStatusBar(HWND hWnd, BOOL bResize)
 
 void UpdateStatusBar(void)
 {
-    HKEY hKeyRoot;
-    LPCWSTR pszKeyPath = GetItemPath(g_pChildWnd->hTreeWnd, 0, &hKeyRoot);
+    HKEY hKeyRoot = NULL;
+    LPCWSTR pszKeyPath = NULL;
+    LPWSTR pszFullPath = NULL;
```

Comment:
```suggestion
    HKEY hKeyRoot;
    LPCWSTR pszKeyPath, pszRootName;
    LPWSTR pszFullPath;
```
These variables get reinitialized unconditionally just below, so there's no point in NULLifying them.